src/ui/Typeahead.tsx

```diff
@@ -118,6 +118,10 @@ const Typeahead: TypeaheadFunc = forwardRef(
             padding: 0.375rem 0.9rem;
           }
 
+          .dropdown-item {
+            border-bottom: 1px solid ${({ theme }) => theme.colors.grey1};
```

Comment:
is there a reason why you use an arrow function?
You could simply write it like
```suggestion
            border-bottom: 1px solid ${theme.colors.grey1};
```

Reply:
Yes theme is not imported but injected via the arrow function (styled-components + own component library magic)